packages/core/test/acceptance/sequence/sequence.acceptance.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright IBM Corp. 2013,2017. All Rights Reserved.
+// Node module: loopback
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {
+  Application, Server, api,
+  OpenApiSpec, ParameterObject,
+  ServerRequest, ServerResponse, parseOperationArgs, writeResultToResponse,
+} from '../../../src';
```

Comment:
we need to get values from index now to make sure coverage runs properly -- ie. remove /src